### PR TITLE
Solves the Void handling BUG in the GroupByKey.

### PR DESCRIPTION
**Pull request by kl0u**

### CODE REVIEW

**Comment by kl0u** [comment]
Also this seems to be ok @mxm! Thanks.

**Comment by mxm** [comment]
@kl0u Could you please clean up the commit history? There are 6 commits.

File: src/test/java/com/dataartisans/flink/dataflow/streaming/GroupByNullKeyTest.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package com.dataartisans.flink.dataflow.streaming;
```
Comment (mxm) [inline src/test/java/com/dataartisans/flink/dataflow/streaming/GroupByNullKeyTest.java]: Missing license.

**Comment by mxm** [comment]
Merged with 0ea22e405615e81a45231b6cfa6cf3f01b00db01.

**Pull request closed without merging**